__tests__/apps.test.js

```diff
@@ -72,4 +73,41 @@ describe('app', () => {
                 });
             });
         });
-});
+    });
+
+
+describe('GET/api/reviews/:review_id', () => {
+    it("gets an id number and receives a review_id object", () => {
+        return request(app)
+        .get("/api/reviews/2")
+              .expect(200)
+              .then(({ body }) => {
+                  expect(body.review.review_id).toBe(2);
+                  expect(body.review).toEqual(
+                    expect.objectContaining({
+                    review_id: expect.any(Number),
+                    title: expect.any(String),
+                    designer: expect.any(String),
+                    owner: expect.any(String),
+                    review_img_url: expect.any(String),
+                    review_body: expect.any(String),
+                    category: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number)
+                }));
+              });
+        });
+    });
+    describe('ERRORS02', () => {
+        test('custom status:404, responds with a custom error when passed a bad user ID', () => {
+            return request(app)
```

Comment:
Good test - you should add another test for /api/reviews/notANumber - i.e. we're testing that when handed a string in place of our expected number parametric endpoint we get the correct error back. Should be a very similar test to this one.